pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -723,6 +726,8 @@ func ManageCSRCABundle(ctx context.Context, lister corev1listers.ConfigMapLister
 	requiredConfigMap, err := resourcesynccontroller.CombineCABundleConfigMaps(
 		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.OperatorNamespace, Name: "csr-controller-ca"},
 		lister,
+		"apiserver-auth",
```

Comment:
this doesn't look right. The producer here is the kube-controller-manager and the consumer is kube-apiserver. I don't think this can simply be assigned to the auth team. There is some claim that the kube-apiserver team owns authentication connections for client certificates signed by the kube-controller-manager for the kube-apiserver, but it feels a stretch.

cc @stlaz

Reply:
Updated